File: extract-pdf.js
import fs from 'fs';
import pdfParseModule from 'pdf-parse';

let pdfParse = pdfParseModule.default || pdfParseModule;

async function extract() {
    try {
        let dataBuffer = fs.readFileSync('public/tarifario-2026.pdf');
        let data = await pdfParse(dataBuffer);
        fs.writeFileSync('public/tarifario-2026-text.txt', data.text);
        console.log("Text extracted to public/tarifario-2026-text.txt");
    } catch (e) {
        console.error("Error:", e);
    }
}
extract();

File: extract-pdf.py
import sys

try:
    import PyPDF2
    with open('public/tarifario-2026.pdf', 'rb') as f:
        reader = PyPDF2.PdfReader(f)
        text = ""
        for page in reader.pages:
            text += page.extract_text() + "\n"
    with open('public/tarifario-2026-text.txt', 'w', encoding='utf-8') as f:
        f.write(text)
    print("Text extracted successfully to public/tarifario-2026-text.txt")
except ImportError:
    print("PyPDF2 not installed. Please run: pip install PyPDF2")
